### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by HT808s**
Hi tried to do a refactoring of the project in order to lay better foundation for the future work

### CODE REVIEW

**Comment by HT808s** [comment]
@sklarsa Let me know what you think of that change and don't hesitate to make suggestions !

File: bloomberg/quote.go [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
 	"fmt"
-	"github.com/aktau/gofinance/fquery"
 	"net/http"
 	"strconv"
 	"strings"
 	"time"
+
+	"github.com/HT808s/gofinance/models"
+
+	"github.com/aktau/gofinance/util"
```
Comment (sklarsa) [inline bloomberg/quote.go]: Should be `github.com/HT808s/gofinance/util`

File: glide.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package: github.com/HT808s/gofinance
+import:
+- package: github.com/fatih/color
+  version: ^1.1.0
+- package: golang.org/x/net
+  subpackages:
+  - html
```
Comment (sklarsa) [inline glide.yaml:7]: Missing `gopkg.in/alecthomas/kingpin.v2`

File: glide.lock [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  version: 30a891c33c7cde7b02a981314b4228ec99380cca
+- name: golang.org/x/net
+  version: 71a035914f99bb58fe82eac0f1289f10963d876c
+  subpackages:
+  - html
+  - html/atom
+- name: golang.org/x/sys
+  version: 8f0908ab3b2457e2e15403d3697c9ef5cb4b57a9
+  subpackages:
+  - unix
```
Comment (sklarsa) [inline glide.lock:24]: Missing `gopkg.in/alecthomas/kingpin.v2`

**Review by sklarsa — CHANGES_REQUESTED** [review]
Just a few small changes to get everything to build... Aside from that, nice work!  I really like the refactor and the cli seems much cleaner than it was before.

**Comment by HT808s** [comment]
@sklarsa Thanks for your valuable review and pointing at me what I missed
Along with the fix I did some minor changes in the workflow 
* Add a `sources`directory on top of yahoofinance and bloomberg packages
* Removed the too "specialized" method `PrettyDisplay` in models/quote.qo to cmd/cli since it contains code which only concerns the terminal app

The code in models should be, if possible, agnostic

**Comment by sklarsa** [comment]
Ya that makes sense to me.  I can make a PR for basic unit tests on the yahoo and bloomberg source APIs (since that's my primary use case at the moment) once this is merged.

**Comment by HT808s** [comment]
I think it's good to merge
Unit tests will be nice !
Actually that's the next topic to take into consideration
I will think at how to improve the overall testability of the project along with it's integration to a CI

**Comment by sklarsa** [comment]
I think we can probably use TravisCI since it's free for open source projects and has solid github integration... unless you have any other inclinations towards something else

**Comment by HT808s** [comment]
TravisCi is ok with me
I'll check on that later

**Pull request merged**